tests/ui/cmse-nonsecure/cmse-nonsecure-entry/params-on-registers.rs

```diff
@@ -3,14 +3,13 @@
 //@ needs-llvm-components: arm
 #![feature(cmse_nonsecure_entry, no_core, lang_items)]
 #![no_core]
-#[lang="sized"]
-trait Sized { }
-#[lang="copy"]
-trait Copy { }
+#[lang = "sized"]
+trait Sized {}
+#[lang = "copy"]
+trait Copy {}
 impl Copy for u32 {}
 
 #[no_mangle]
-#[cmse_nonsecure_entry]
-pub extern "C" fn entry_function(_: u32, _: u32, _: u32, d: u32) -> u32 {
+pub extern "C-cmse-nonsecure-entry" fn entry_function(_: u32, _: u32, _: u32, d: u32) -> u32 {
     d
 }
```

Comment:
it would be cool if we could check that this function actually uses the `bxns` instruction for its return, like e.g. here for x86 https://github.com/rust-lang/rust/blob/master/tests/codegen/option-as-slice.rs. Does that even work on arm? I can't get it to work/fail locally, the `CHECK` comments just seem to be ignored

Reply:
figured out how to do this